src/Auth0Client.ts

```diff
@@ -625,10 +625,7 @@ export default class Auth0Client {
     });
 
     if (!cache || !cache.refresh_token) {
-      throw new GenericError(
-        'missing_refresh_token',
-        'No refresh token is available to fetch a new access token. The user should be reauthenticated.'
-      );
+      return await this._getTokenFromIFrame(options);
```

Comment:
This is the crux of the fix. Instead of throwing an error if there's no refresh token in the cache, we fall back to the legacy iframe method of getting a new access token.